lessons/taquito/05/05.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: How much does it cost?
+chapter: Chapter 5
+slug: module-04/chapter-05
+filterBy: lesson-4
+editor:
+  language: javascript
+  startingCode: |
+    Tezos.estimate.originate({
```

Comment:
Example code.

`println(`Estimating the contract origination : `);
  Tezos.estimate.originate({
    code: genericMultisigJSONfile,
    storage: {
      stored_counter: 0,
      threshold: 1,
      keys: ['edpkuLxx9PQD8fZ45eUzrK3BhfDZJHhBuK4Zi49DcEGANwd2rpX82t']
    }
  })
  .then (originationOp => {
    println(`burnFeeMutez : ${originationOp.burnFeeMutez}, 
    gasLimit : ${originationOp.gasLimit}, 
    minimalFeeMutez : ${originationOp.minimalFeeMutez}, 
    storageLimit : ${originationOp.storageLimit}, 
    suggestedFeeMutez : ${originationOp.suggestedFeeMutez}, 
    totalCost : ${originationOp.totalCost}, 
    usingBaseFeeMutez : ${originationOp.usingBaseFeeMutez}`)
    })
  .catch(error => println(`Error: ${JSON.stringify(error, null, 2)}`));`

Currently, it's missing error block.